src/java/org/apache/cassandra/db/lifecycle/View.java

```diff
@@ -18,6 +18,9 @@
 package org.apache.cassandra.db.lifecycle;
 
 import java.util.*;
+import java.util.stream.Collector;
+import java.util.stream.Collectors;
```

Comment:
nit: unused import. If you remove the collect method I think you could delete all of these